src/private/traits.rs

```diff
@@ -46,6 +46,6 @@ pub trait IntoBits<T> {
 ///
 /// Implemented by primitive specifier types.
 #[doc(hidden)]
-pub trait FromBits<T> {
-    fn from_bits(bits: Bits<T>) -> Self;
+pub trait FromBits<BASE, FACE> {
+    fn from_bits(bits: Bits<BASE>) -> FACE;
 }
```

Comment:
I wonder if it wouldn't be simpler if we could instead define this trait like this:
```rust
pub trait FromBits<T>
where
    T: Specifier,
{
    fn from_bits(bits: Bits<<T as Specifier>::Base>) -> <T as Specifier>::Face;
}
```

Reply:
Hm, after poking at it a bit, what if `from_bits` a method directly on the `Specifier` trait? We know that any given `Specifier` is only going to have a single canonical `Base` representation, and we only need to convert it to a single value (`GetterReturn`), so it seems like that might be even simpler.

I think we can even change the `from_bits` signature to not wrap the incoming value with `Bits` (Originally, I was having trouble importing `Bits` from `define_specifier`, since we'd need to implement `FromBits` for each one of those specifiers, but I think it was just me being dumb. That's why I went down that path though)

Reply:
I have a branch here that crudely prototypes that: https://github.com/lkolbly/modular-bitfield/tree/lkolbly/remove-frombits

Reply:
To be fair: This crate is a bit old already and I was also working on some major updated on a local branch that also completely restructure the `Specifier` trait. So whatever you will do in this branch is going to be merged to this crate and I will release a version so that you can depend on your own work but it might be updated soon when my own updates are coming in that overhaul the entire backend.

Reply:
Okay, makes sense. I'm inclined to drop this comment then, and leave `FromBits` with two parameters in this PR.